privaterelay/settings.py

```diff
@@ -183,6 +183,8 @@
 MAX_MINUTES_TO_VERIFY_REAL_PHONE = config(
     "MAX_MINUTES_TO_VERIFY_REAL_PHONE", 5, cast=int
 )
+MAX_TEXTS_PER_BILLING_CYCLE = config("MAX_TEXTS_PER_BILLING_CYCLE", 75, cast=int)
+MAX_MINUTES_PER_BILLING_CYCLE = config("MAX_MINUTES_PER_BILLING_CYCLE", 50, cast=int)
```

Comment:
Note: this may need to be "per month" since there will be annual billing cycles soon.